launcher/net/ByteArraySink.h

```diff
@@ -58,6 +58,7 @@ class ByteArraySink : public Sink {
             qWarning() << "ByteArraySink did not initialize the buffer because it's not addressable";
         if (initAllValidators(request))
             return Task::State::Running;
+        m_fail_reason = "failed to initialize validators";
```

Comment:
```suggestion
        m_fail_reason = "Failed to initialize validators";
```
Same for the rest of the fail reasons